babel.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   presets: [
     ["@babel/preset-env", { targets: { node: "current" } }],
-    "@babel/preset-typescript"
-  ]
+    "@babel/preset-typescript",
+    "preact"
+  ],
+  plugins: [["@babel/plugin-proposal-decorators", { legacy: true }]]
```

Comment:
oof, another thing to work on is switching from the many-versions-obsolete decorators to the stage 3 ones (or none at all)

Reply:
going to limit the scope of this PR but I'll leave in a comment

Reply:
100% should be in a different PR :-)